simbad/command_line/simbad_database.py

```diff
@@ -163,6 +189,151 @@ def create_lattice_db(database):
     np.savez_compressed(database, niggli_data)
 
 
+def create_contaminant_db(database, nproc=2, submit_qtype=None, submit_queue=False):
+    """Create a contaminant database
+
+    Parameters
+    ----------
+    database : str
+       The path to the database folder
+    nproc : int, optional
+       The number of processors [default: 2]
+    submit_qtype : str
+       The cluster submission queue type - currently support SGE and LSF
+    submit_queue : str
+       The queue to submit to on the cluster
+
+    Raises
+    ------
+    RuntimeError
+        dimple.contaminants.prepare module not available
+    RuntimeError
+       Windows is currently not supported
+    """
+
+    try:
+        import dimple.contaminants.prepare
```

Comment:
Could you not check for the `dimple` version? I think that would be more explicit